crypto/asn1/x_long.c

```diff
@@ -56,11 +56,30 @@ static void long_free(ASN1_VALUE **pval, const ASN1_ITEM *it)
     *(long *)pval = it->size;
 }
 
+static int num_bits_ulong(unsigned long value)
```

Comment:
The point is still that we now have two function performing the same type of calculation, albeit one being more efficient than the other.  Wouldn't it be better to refactor it so we have *one* general function that takes a `uint64_t`, that's called below and by `BN_num_bits_word()`.  I.e. *one* place that counts bits instead of two?